configs/pontoon.toml

```diff
@@ -234,6 +227,9 @@ locales = [
         "nl",
         "sv-SE",
     ]
+[[paths]]
+    reference = "en/firefox/whatsnew/whatsnew-developer-mdnplus"
```

Comment:
Pretty sure it's not expected for this to be missing `.ftl`.

Reply:
Doh. Fixed in https://github.com/mozilla/bedrock/pull/11764. Sorry about that.